lib/ansible/modules/fetch.py

```diff
@@ -56,6 +56,18 @@
     - If using multiple hosts with the same filename, the file will be overwritten for each host.
     type: bool
     default: no
+  encrypt:
+    version_added: '2.15'
+    description:
+    - If enabled, the local file will be encrypted into a vault. In this case, the vault_id parameter is required.
```

Comment:
Is the last sentance true, seems like the `default: default` covers this and is not actually required.

Reply:
Correct, missed this part in latest edit. Fixed.